doc/sips/README.md

```diff
@@ -3,3 +3,10 @@ People wishing to submit Sia Improvement Proposals (SIPs) should first propose t
 Having a SIP here does not make it a formally accepted standard until its status becomes Final or Active.
 
 Those proposing changes should consider that ultimately consent may rest with the consensus of the Sia users.
+
+| Number  | Layer   | Title                   | Owner     | Status  |
+| :-----: | :-----: | :---------------------: | :-------: | :-----: |
+| [1][1]  | renter  | Store renter metadata remotely | [@starius][starius] | draft |
```

Comment:
please adjust readme spacing to have aligned columns

Reply:
Done